src/PayplugWoocommerceHelper.php

```diff
@@ -486,6 +486,9 @@ public static function get_account_data_from_options()
 			$account['bancontact'] = !empty($options['bancontact']) ? $options['bancontact'] : '';
 			$account['american_express'] = !empty($options['american_express']) ? $options['american_express'] : '';
 
+			if( !isset($account['payment_methods']['apple_pay']) )
```

Comment:
I know we've on the code many places without `{` done by everyone, but we should always use them.